client/my-sites/stats/stats-module/all-time-nav.jsx

```diff
@@ -37,14 +33,19 @@ export const StatsModuleSummaryLinks = props => {
 		}
 	};
 
+	const recordStats = ( item ) => {
+		props.recordGoogleEvent( 'Stats', `Clicked Summary Link: ${ path } ${ item.stat }` );
```

Comment:
My bad, looks like `path` is different from `item.path`. Based on the diff it did not occur to me that it's in the same scope. Carry on :)